burr/integrations/haystack.py

```diff
@@ -39,8 +40,10 @@ def __init__(
         Use a mapping {state_field: socket} to rename Haystack output sockets (see example).
         :param name: Name of the action. Can be set later via ``.with_name()``
         or in ``ApplicationBuilder.with_actions()``.
-        :param bound_params: Parameters to bind to the `Component.run()` method.
-
+        :param bound_params: Parameters to bind to the ``Component.run()`` method.
+        :param is_warm: If True, try to call ``Component.warm_up()`` if it exists.
```

Comment:
This doesn't match the params? Think you just want to change the param name in the docstring.